servers/team_allocation/tease/teaseDTO/skillResponse.go

```diff
@@ -0,0 +1,38 @@
+package teaseDTO
+
+import (
+	"github.com/google/uuid"
+	db "github.com/ls1intum/prompt2/servers/team_allocation/db/sqlc"
+)
+
+type SkillResponse struct {
+	ID          uuid.UUID `json:"id"`
+	Proficiency string    `json:"proficiency"`
+}
```

Comment:
Since this corresponds to "StudentSkill" in Tease and there is also another type "Skill", for consistency, I would call this "StudentSkill" here, too.

```suggestion
type StudentSkillResponse struct {
	ID          uuid.UUID `json:"id"`
	Proficiency string    `json:"proficiency"`
}
```

Reply:
I disagree here. This is not supposed to model the Prompt Data model but to model the objects required from Tease. 

Choosing here StudentSkill instead of skill shows that this is not the same and StudentSkill is the name used by tease. 

Using Skill would be counter intuitive in my understanding as the Prompt Skill does not have a "proficiency"

Reply:
Fair point